pymc/tests/test_data.py

```diff
@@ -695,16 +695,13 @@ def test_common_errors(self):
         assert "Double Ellipsis" in str(e.value)
 
     def test_mixed1(self):
-        with pm.Model():
-            data = np.random.rand(10, 20, 30, 40, 50)
-            mb = pm.Minibatch(data, [2, None, 20, Ellipsis, 10])
-            pm.Normal("n", observed=mb, total_size=(10, None, 30, Ellipsis, 50))
+        from pymc.distributions.logprob import joint_logp
 
-    def test_mixed2(self):
         with pm.Model():
-            data = np.random.rand(10, 20, 30, 40, 50)
-            mb = pm.Minibatch(data, [2, None, 20])
-            pm.Normal("n", observed=mb, total_size=(10, None, 30))
+            data = np.random.rand(10, 20)
+            mb = pm.Minibatch(data, batch_size=5)
+            v = pm.Normal("n", observed=mb, total_size=10)
+            assert joint_logp(v) is not None, "Check index is allowed in graph"
```

Comment:
`joint_logp` is deprecated. You can test that it does not fail by just calling `m.logp()`